regtests/0243_sshort/sshort.adb

```diff
@@ -186,14 +186,18 @@ begin
 
    Peer.Send (Sample (1 .. Last));
 
+   if Peer.Receive /= (1 => 11) then
+      Ada.Text_IO.Put_Line ("Unexpected responce");
```

Comment:
`Ada.` can be removed.

Reply:
First issue fixed in separate last commit.
The second one by modify of the same commit.